src/components/nodes/textAnnotation/textAnnotation.vue

```diff
@@ -88,7 +93,7 @@ export default {
       });
 
       this.paper.once('render:done', () => {
-        this.setElementHeight(height, bounds.height, text);
+        this.shape.resize(this.nodeWidth, this.calculateNewHeight(height, text, bounds.height));
```

Comment:
I decided to refactor this as I worked on extracting the padding into a variable. The problem with the previous method was that it combined a command and a query IMO.